kiwi/solverimpl.h

```diff
@@ -650,22 +643,20 @@ class SolverImpl
 	is returned.
 
 	*/
-	Symbol getDualEnteringSymbol( const Row& row )
+	Symbol getDualEnteringSymbol( const Row& row ) const
```

Comment:
This is an API incompatibility. Not sure what the protocol for this project is, but it's worth pointing out that this might break some users (even though I believe the change is correct).

This goes for the other couple of places that change the method signature.

@MatthieuDartiailh might want to make sure this isn't going to cause problems.

Reply:
Making a non-const function const is not incompatible, as there's no case where you would be able to call a non-const function but not a const one (it would be incompatible if I removed a const keyword)

Reply:
I just realized that this is a non-issue as `getDualEnteringSymbol` is private and thus not part of the API.